lib/adapter.js

```diff
@@ -6972,24 +6961,18 @@ function Adapter(options) {
                                 // If alias
                                 if (id.startsWith(ALIAS_STARTS_WITH)) {
                                     const aliasObj = objs[id];
-
-                                    count++;
-                                    this._addAliasSubscribe(aliasObj, aliasPattern, () => {
-                                        if (!--count) {
-                                            if (pattern === '*') {
-                                                adapterStates.subscribeUser(pattern, callback);
-                                            } else {
-                                                callback && callback();
-                                            }
-                                        }
-                                    });
+                                    promises.push(new Promise(resolve => this._addAliasSubscribe(aliasObj, aliasPattern, resolve)));
```

Comment:
could do the same here, just make the callback method `async`.

Reply:
also just alias subscribe, nothing which could overload our db